tools/lsp/language/completion.rs

```diff
@@ -341,6 +341,8 @@ pub(crate) fn completion_at(
             ("export component", "export component ${1:ExportedComponent} {\n    $0\n}"),
             ("export struct", "export struct ${1:Name} {\n    $0\n}"),
             ("export global", "export global ${1:Name} {\n    $0\n}"),
+            ("new window", "export component ${1:MyMainWindow} inherits Window {\n    $0\n}"),
+            ("new dialog", "export component ${1:MyDialog} inherits Dialog {\n    $0\n}"),
```

Comment:
We should not add this if the editor don't support snippet.  For the others string, the first string will be completed if the editor don't support snippets, and that's ok. But here, "new window" makes no sense to complete literally)